lib/feed.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
-<?xml-stylesheet type="text/xml" href="{{ url_base }}/feed.xslt.xml"?>
```

Comment:
Whoops! Thanks for noticing.

For the benefit of the changelog, could you punt this in a separate PR please?